Dockerfile.alpine

```diff
@@ -0,0 +1,17 @@
+FROM python:3.12-alpine3.20
+
+RUN apk upgrade --update && apk add --no-cache \
+  bash \
+  git \
+  cargo \
+  linux-headers
+
+WORKDIR /app
+
+COPY requirements.txt .
+RUN pip install --no-cache-dir --no-dependencies -r requirements.txt
+
+COPY src gulp_cfg_template.json MANIFEST.in pyproject.toml .
+RUN --mount=source=.git,target=.git,type=bind pip install -e .
```

Comment:
i'm not sure why you need to add `--mount=source=.git,target=.git,type=bind` but it fails otherwise 🤷🏻